front/src/components/CTF/CardList.vue

```diff
@@ -1,58 +1,208 @@
 <template>
-  <div class="row q-col-gutter-lg justify-evenly">
-    <div
-      v-for="(column, idx) in columns"
-      :key="idx"
-      class="column col q-gutter-lg justify-start align-center"
+  <div class="row justify-center scroll-anchor">
+    <q-btn color="primary" @click="fetchMore(shortDate.prev(ctfByMonth[0]))">
+      Load previous ctfs
+    </q-btn>
+  </div>
+  <div
+    v-for="group of ctfByMonth"
+    :key="shortDate.toId(group)"
+    class="full-size"
+    :data-date="shortDate.toId(group)"
+  >
+    <IntersectionObserver
+      :options="{ rootMargin: '200px' }"
+      class="column gap-lg"
+      @show.once="fetchMoreCtfs(group)"
     >
-      <q-intersection
-        v-for="ctf in column"
+      <div class="row items-center gap-md scroll-anchor">
+        <q-separator class="col" />
+        <div class="col-auto col text-h6 q-py-sm">
+          {{
+            date
+              .buildDate({ year: group.year, month: group.month })
+              .toLocaleDateString(undefined, { year: 'numeric', month: 'long' })
```

Comment:
Here you use `toLocaleDateString` which will translate the current month to the browser defined language. This makes the UI inconsistent: we use English everywhere except here. I would suggest to keep it English here too.